src/shared/utils/urlGenerator.ts

```diff
@@ -52,7 +53,7 @@ export const getAppletPageRegexp = (path?: string) =>
 export const getAppletActivityPageRegexp = (path?: string) =>
   path ? `${ACTIVITY_PAGE_REGEXP_STRING}\\/${path}` : ACTIVITY_PAGE_REGEXP_STRING;
 export const getAppletActivityFlowPageRegexp = (path?: string) =>
-  path ? `${ACTIVITY_FLOW_PAGE_REGEXP_STRING}\\/${path}` : ACTIVITY_PAGE_REGEXP_STRING;
+  path ? `${ACTIVITY_FLOW_PAGE_REGEXP_STRING}\\/${path}` : ACTIVITY_FLOW_PAGE_REGEXP_STRING;
```

Comment:
👀

Reply:
seemed to be a mistake but we didn't find it because this code was never reached :)